startup_runners/outer_startup_runner.c

```diff
@@ -35,7 +35,7 @@ main(int argc, char **argv)
 {
     int result, num_per_core_data;
 
-    if (argc != 2) {
+    if (argc < 2) {  // krun passs more args, which are ignored here.
```

Comment:
Ah, I see what you mean. Can we simplify the comment to something like "Ensure we get the arguments we need, but we're OK with extra arguments being passed (such as by Krun)". [OK, that's a rubbish comment, but you get the idea.]